contracts/cw20-merkle-airdrop/src/msg.rs

```diff
@@ -0,0 +1,170 @@
+use crate::ContractError;
+use cosmwasm_std::{from_slice, Binary, Uint128};
+use cw_utils::{Expiration, Scheduled};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct InstantiateMsg {
+    /// Owner if none set to info.sender.
+    pub owner: Option<String>,
+    pub credits_address: Option<String>,
+    pub reserve_address: Option<String>,
```

Comment:
if this contract is initialized the same way the credits contract is (in genesis), then this options are not necessary since we will know the addresses for sure

Reply:
imo can be done in a separate PR. marking it with this msg to create a list of such complaints a bit later